src/containers/chat/active-messenger/tests/active-messenger.spec.js

```diff
@@ -26,8 +19,7 @@ const useStateSpy = jest.spyOn(React, 'useState');
 const mockUseDispatch = jest.spyOn(redux, 'useDispatch');
 const mockUseSelector = jest.spyOn(redux, 'useSelector');
```

Comment:
Replace this with jest mocking react redux

Reply:
sorry, missed that)
done